test/index.test.js

```diff
@@ -1,9 +0,0 @@
-'use strict';
-
-const assert = require('chai').assert;
-
-describe('index test', () => {
```

Comment:
If it's too difficult to test in its current form, then we should refactor it so that it's testable.